.yamllint

```diff
@@ -0,0 +1,5 @@
+---
+extends: default
+
+rules:
+  line-length: disable
```

Comment:
Is there any way to set the line-length instead of just disabling this rule?

Reply:
Yep, we can adjust to the desired length but better to disable as there are commands or paths that can break the rule (Not at this role but also good to have)